wallet/wallet.c

```diff
@@ -3143,7 +3143,8 @@ wallet_payment_list(const tal_t *ctx,
 						  ", partid"
 						  ", local_offer_id"
 						  " FROM payments"
-						  " WHERE payment_hash = ?;"));
+						  " WHERE payment_hash = ?"
+						  " ORDER BY id;"));
```

Comment:
Do you think that sort by timestamp it is a good idea here?

Reply:
id is the order it was placed in the db, which is probably right here.